cmp/form/FormModel.js

```diff
@@ -221,5 +222,6 @@ export class FormModel {
 
     destroy() {
         XH.safeDestroy(this.fieldList);
```

Comment:
Does `@managed` not work on getters? If not, should we provide some kind of `markManagedProp` similar to `markManaged` for dynamically adding a prop to be managed?